website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md

```diff
@@ -0,0 +1,114 @@
+
+This Migration Guide will show you **how to create a custom Fabric component or a TurboModule** with a step-by-step approach. The guide will also show you how to **adapt your existing app or library** to use the New Architecture.
+
+Moreover, we would like to remind you the brand-new [Architecture section](/architecture/overview) of our website. There you can find several in-depth articles and explanation of the React Native internals. Specifically, [the Fabric section](/architecture/fabric-renderer) can help you understand the rendering pipeline in the New Architecture world.
+
+Finally, please consider **sharing your feedback** to this documentation material [on the working group](https://github.com/reactwg/react-native-new-architecture/discussions/7). We’re constantly looking for developer’s opinion, and we want to make sure we’re delivering the content that you find most useful.
+
+Over the next months, we will look into refining and adding more documentation to help you further.
+
+### The New Architecture Template
+
+React Native **0.68.0** is close to release. This version of React Native marks a crucial milestone in the New Architecture Rollout as it’s the first version to include an **opt-in switch** in the **new app template.**
+
+This means that you will be able to try the New Architecture **by changing one line** in the template. We also added extensive **comments and documentation** to the template to make sure you don’t need extra reading to use it out of the box. We hope this will help you adopt the New Architecture by **reducing the amount of code** your have to write.
+
+<!-- alex ignore simple -->
+
+In the next releases, we will keep on updating the template to make it even more streamlined and simple to use.
+
+To enable the New Architecture on either platform, you can:
+
+- On iOS, run `RCT_NEW_ARCH_ENABLED=1 pod install` inside the `ios` folder.
+- On Android, set the `newArchEnabled` property to `true` by **either**:
+  - Changing the corresponding line inside the `android/gradle.properties` file.
+  - Set an environment variable `ORG_GRADLE_PROJECT_newArchEnabled=true`
+  - Invoke Gradle with `-PnewArchEnabled=true`
+
+Then you can **run your app** with `yarn react-native run-android` or `run-ios` and you’ll be running using Fabric and TurboModules enabled.
+
+Please consider trying this new template, and [report any bug or unexpected behavior](https://github.com/reactwg/react-native-new-architecture/discussions/5) that you might face. Over the last months we worked hard to fix bugs and build failures that would have been **hard to catch** without the constant community feedback and testing.
+
+### The 3rd-party Libraries Ecosystem
+
+We could not migrate to the New Architecture without the full support of **3rd-party libraries author & maintainers**.
```

Comment:
```suggestion
We can not migrate to the New Architecture without the full support of **3rd-party libraries author & maintainers**.
```
nit: this should not be using the past for "could", right? It's not like the new arch is out and everyone has migrated to it.

Or maybe it should be even more clear, along the lines of:
> The community won't be able to migrate to the New Architecture without the support of their fellow **libraries author & maintainers**.

Reply:
Fixed 👍